### DISCUSSION TITLE
How to create Workflow Template Request?

### DISCUSSION THREAD

**Discussion by TomerAviassafCider** [argoproj/argo-workflows#9960] [Q&A]
Hey!
I would like to create a Workflow **Template** Request by using IoArgoprojWorkflowV1alpha1WorkflowTemplateCreateRequest class.

`from argo_workflows.model.io_argoproj_workflow_v1alpha1_workflow_template_create_request import IoArgoprojWorkflowV1alpha1WorkflowTemplateCreateRequest`

For example, **Workflow** creation using `IoArgoprojWorkflowV1alpha1WorkflowCreateRequest`:
```
api_client = argo_workflows.ApiClient(configuration)
api_instance = workflow_service_api.WorkflowServiceApi(api_client)
api_response = api_instance.create_workflow(
    namespace="default",
    body=IoArgoprojWorkflowV1alpha1WorkflowCreateRequest(workflow=manifest, _check_type=False),
    _check_return_type=False)
```

Thank you,
Tomer

**Comment by terrytangyuan** [comment]
You are using the wrong API `api_instance.create_workflow`

**Reply by TomerAviassafCider** [reply]
Sure, I used `api_instance.create_workflow` for the workflow creation example.
Which API should i use?

**Reply by leesenchiu** [reply]
> Sure, I used `api_instance.create_workflow` for the workflow creation example. Which API should i use?
 do you fix this problem?
